addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationEnvironment.java

```diff
@@ -162,6 +170,16 @@ public Context parseContextData(
         }
         Context context = session.getNewContext(name);
         context.addIncludeInContextRegex(url + ".*");
+        if (includeRegexes != null) {
+            for (Object regex : includeRegexes) {
+                context.addIncludeInContextRegex(replaceVars(regex.toString()));
```

Comment:
We'll need some error handling - eg what happens if they specify invalid regex expressions or something other than an ArrayList?